### PR TITLE
Template update for nf-core/tools v3.1.0 

### PR DESCRIPTION
**Pull request by LilyAnderssonLee**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/meta-val/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .nf-core.yml [hunk 1]
```diff
@@ -1,9 +1,11 @@
-bump_version: null
 lint:
   files_exist:
     - assets/nf-core-meta-val_logo_light.png
+    - assets/nf-core-metaval_logo_light.png
     - docs/images/nf-core-meta-val_logo_light.png
     - docs/images/nf-core-meta-val_logo_dark.png
+    - docs/images/nf-core-metaval_logo_light.png
+    - docs/images/nf-core-metaval_logo_dark.png
```
Comment (maxulysse) [inline .nf-core.yml]: how many versions of the logo do you have?

Comment (maxulysse) [reply .nf-core.yml]: Mainly thinking about the `meta-val` vs `metaval`

File: CITATIONS.md [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 > Ewels PA, Peltzer A, Fillinger S, Patel H, Alneberg J, Wilm A, Garcia MU, Di Tommaso P, Nahnsen S. The nf-core framework for community-curated bioinformatics pipelines. Nat Biotechnol. 2020 Mar;38(3):276-278. doi: 10.1038/s41587-020-0439-x. PubMed PMID: 32055031.
 
+## [KrakenTools](https://www.nature.com/articles/s41596-022-00738-y)
+
+> Lu J, Rincon N, Wood DE, et al. Metagenome analysis using the Kraken software suite. Nat Protoc. 2022;17(12). doi:10.1038/s41596-022-00738-y
```
Comment (maxulysse) [inline CITATIONS.md]: that should stay in the pipeline tools, no?

**Review by maxulysse — APPROVED** [review]
LGTM

File: CITATIONS.md (hunk 2 above)
Comment (LilyAnderssonLee) [reply CITATIONS.md]: Yes, you are right!

File: .nf-core.yml (hunk 1 above)
Comment (LilyAnderssonLee) [reply .nf-core.yml]: I’m keeping `meta-val` for now, but I’m confused between them. In practice it’s not good to have a dash in the repo name. I’m considering renaming `meta-val` to either `metaeval` or `medieval`, as suggested by @jfy133.

File: assets/methods_description_template.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-id: "genomic-medicine-sweden-metaval-methods-description"
+id: "genomic-medicine-sweden-meta-val-methods-description"
```
Comment (maxulysse) [inline assets/methods_description_template.yml:1]: here you have `meta-val`

File: docs/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# genomic-medicine-sweden/meta-val: Documentation
+# genomic-medicine-sweden/metaval: Documentation
```
Comment (maxulysse) [inline docs/README.md:1]: here you have `metaval`

File: .nf-core.yml (hunk 1 above)
Comment (maxulysse) [reply .nf-core.yml]: I'd just go for consistency, you have some `metaval` in your codebase, and some `meta-val` mixed in

Comment (LilyAnderssonLee) [reply .nf-core.yml]: I am going to merge this PR and try to fix the consistency of pipeline name in the separate PR.

File: assets/methods_description_template.yml (hunk 3 above)
Comment (LilyAnderssonLee) [reply assets/methods_description_template.yml:1]: Well, I'm going to open a PR to correct the name of my pipeline. This is really annoying to me

**Pull request merged**